modules/admin_manual/pages/appliance/configuration/clamav.adoc

```diff
@@ -86,5 +96,5 @@ sudo freshclam
 
 [WARNING]
 ====
-When the app is enabled — but is not configured or has an incorrect configuration — it will reject **all** uploads for the entire instance. To avoid this situation, use `{occ-command-example-prefix} config:app:set files_antivirus whatever_key value` to configure the app correctly.
+When the app is enabled — but is not configured or has an incorrect configuration — it will reject **all** uploads for the entire instance. To avoid this situation, make sure the ClamAV service is running and you have the execution mode correctly configured  in ownCloud.
```

Comment:
Please remove the double space between "configured" and "in".

Reply:
removed